Error in compiling a C program for ALPHA: " no .eh_frame_hdr table will be created."

I have build the cross compiler for ALPHA using crosstool-ng. I am not able to compile any C/C++ code. It is throwing the following error:

:~/Desktop$ alphaev4-unknown-linux-gnu-g++ -static new1.cpp -o hello
/home/iiitd/local/x-tools/alphaev4-unknown-linux-gnu/lib/gcc/alphaev4-unknown-linux-gnu/6.3.1/../../../../alphaev4-unknown-linux-gnu/bin/ld: error in /home/iiitd/local/x-tools/alphaev4-unknown-linux-gnu/lib/gcc/alphaev4-unknown-linux-gnu/6.3.1/crtend.o(.eh_frame); no .eh_frame_hdr table will be created.

- crosstool-NG version: crosstool-ng-1.23.0.rc2
- Host OS: ubuntu 16.04 LTS, 64-bit

Okay @stilor . I came across this link "https://github.com/crosstool-ng/crosstool-ng/issues/370" which had the same issue.. According to the issue, I tried to build with crosstool-ng-1.15.0. The steps are as follows:

$ mkdir -p ~/src/gem2/toolchain  
$ cd ~/src/gem2/toolchain  
$  wget http://crosstool-ng.org/download/crosstool-ng/crosstool-ng-1.15.0.tar.bz2  
$ tar xjf crosstool-ng-1.15.0.tar.bz2  
$ cd crosstool-ng-1.15.0/  
$ ./configure -prefix=/home/iiitd/local/gem2-crosstool-ng  
--------------------------------
---------------------------------
configure: error: could not find GNU libtool >= 1.5.26
$ sudo apt-get install libtool-bin
$ ./configure -prefix=/home/iiitd/local/gem2-crosstool-ng
$ make
**Makefile:114: *** Recursion detected, bailing out....  Stop.
Makefile:119: recipe for target 'build' failed
make: *** [build] Error 2**

I am attaching the config.log.  
https://drive.google.com/file/d/0B3MslpubFyURMTBjcjhFZlJXeWM/view?usp=sharing

--- a/patches/gcc/4.8.5/111-alpha-bad-eh_frame.patch
+++ b/patches/gcc/4.8.5/111-alpha-bad-eh_frame.patch
@@ -1,0 +1,13 @@
+https://gcc.gnu.org/bugzilla/show_bug.cgi?id=80037
+
+diff --git a/libgcc/config/alpha/t-alpha b/libgcc/config/alpha/t-alpha
+index 0b6ffb1..0c2f840 100644
+--- a/libgcc/config/alpha/t-alpha
++++ b/libgcc/config/alpha/t-alpha
+@@ -1,2 +1,6 @@
+ # This is a support routine for longlong.h, used by libgcc2.c.
+ LIB2ADD += $(srcdir)/config/alpha/qrnnd.S
++
++# When GAS-generated unwind tables are created, they get created
++# after the __FRAME_END__ terminator, which causes an ld error.
++CRTSTUFF_T_CFLAGS = -fno-unwind-tables

--- a/patches/gcc/4.9.4/111-alpha-bad-eh_frame.patch
+++ b/patches/gcc/4.9.4/111-alpha-bad-eh_frame.patch
@@ -1,0 +1,13 @@
+https://gcc.gnu.org/bugzilla/show_bug.cgi?id=80037
+
+diff --git a/libgcc/config/alpha/t-alpha b/libgcc/config/alpha/t-alpha
+index 0b6ffb1..0c2f840 100644
+--- a/libgcc/config/alpha/t-alpha
++++ b/libgcc/config/alpha/t-alpha
+@@ -1,2 +1,6 @@
+ # This is a support routine for longlong.h, used by libgcc2.c.
+ LIB2ADD += $(srcdir)/config/alpha/qrnnd.S
++
++# When GAS-generated unwind tables are created, they get created
++# after the __FRAME_END__ terminator, which causes an ld error.
++CRTSTUFF_T_CFLAGS = -fno-unwind-tables

--- a/patches/gcc/5.4.0/111-alpha-bad-eh_frame.patch
+++ b/patches/gcc/5.4.0/111-alpha-bad-eh_frame.patch
@@ -1,0 +1,13 @@
+https://gcc.gnu.org/bugzilla/show_bug.cgi?id=80037
+
+diff --git a/libgcc/config/alpha/t-alpha b/libgcc/config/alpha/t-alpha
+index 0b6ffb1..0c2f840 100644
+--- a/libgcc/config/alpha/t-alpha
++++ b/libgcc/config/alpha/t-alpha
+@@ -1,2 +1,6 @@
+ # This is a support routine for longlong.h, used by libgcc2.c.
+ LIB2ADD += $(srcdir)/config/alpha/qrnnd.S
++
++# When GAS-generated unwind tables are created, they get created
++# after the __FRAME_END__ terminator, which causes an ld error.
++CRTSTUFF_T_CFLAGS = -fno-unwind-tables

--- a/patches/gcc/6.3.0/111-alpha-bad-eh_frame.patch
+++ b/patches/gcc/6.3.0/111-alpha-bad-eh_frame.patch
@@ -1,0 +1,13 @@
+https://gcc.gnu.org/bugzilla/show_bug.cgi?id=80037
+
+diff --git a/libgcc/config/alpha/t-alpha b/libgcc/config/alpha/t-alpha
+index 0b6ffb1..0c2f840 100644
+--- a/libgcc/config/alpha/t-alpha
++++ b/libgcc/config/alpha/t-alpha
+@@ -1,2 +1,6 @@
+ # This is a support routine for longlong.h, used by libgcc2.c.
+ LIB2ADD += $(srcdir)/config/alpha/qrnnd.S
++
++# When GAS-generated unwind tables are created, they get created
++# after the __FRAME_END__ terminator, which causes an ld error.
++CRTSTUFF_T_CFLAGS = -fno-unwind-tables

--- a/patches/gcc/7.1.0/111-alpha-bad-eh_frame.patch
+++ b/patches/gcc/7.1.0/111-alpha-bad-eh_frame.patch
@@ -1,0 +1,13 @@
+https://gcc.gnu.org/bugzilla/show_bug.cgi?id=80037
+
+diff --git a/libgcc/config/alpha/t-alpha b/libgcc/config/alpha/t-alpha
+index 0b6ffb1..0c2f840 100644
+--- a/libgcc/config/alpha/t-alpha
++++ b/libgcc/config/alpha/t-alpha
+@@ -1,2 +1,6 @@
+ # This is a support routine for longlong.h, used by libgcc2.c.
+ LIB2ADD += $(srcdir)/config/alpha/qrnnd.S
++
++# When GAS-generated unwind tables are created, they get created
++# after the __FRAME_END__ terminator, which causes an ld error.
++CRTSTUFF_T_CFLAGS = -fno-unwind-tables

